common/enablers/enableEpochsHandler.go

```diff
@@ -1,16 +1,122 @@
 package enablers
 
 import (
+	"runtime/debug"
 	"sync"
 
+	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-go/common"
 	"github.com/multiversx/mx-chain-go/config"
 	"github.com/multiversx/mx-chain-go/process"
 	logger "github.com/multiversx/mx-chain-logger-go"
 )
 
 var log = logger.GetOrCreate("common/enablers")
 
+var allFlagsDefined = map[core.EnableEpochFlag]struct{}{
+	common.SCDeployFlag:                                       {},
```

Comment:
maybe add here the handler for checking if the flag is enabled in the given epoch as value?

Reply:
updated as discussed